communication/src/allocator/mod.rs

```diff
@@ -32,6 +32,12 @@ pub trait AllocateBuilder : Send {
     fn build(self) -> Self::Allocator;
 }
 
+use std::any::Any;
+
+/// A type that can be sent along an allocated channel.
+pub trait Exchangeable : Send+Any+Bytesable+'static { }
```

Comment:
Maybe move to crate's root?

Reply:
Could re-publish there, but I think all of the uses are in this module (and in other crates).